gbmgeometry/gbm_detector.py

```diff
@@ -58,20 +60,77 @@ def update_position(self, quaternion, sc_pos=None, time=None):
                                                sc_pos_Y=scy,
                                                sc_pos_Z=scz,
                                                ))
-
+        self._x = SkyCoord(lon=0 * u.deg,
```

Comment:
You need to add descriptive comments so that we will know what is going on in the future. I also need to do the same, but it is good practice to go ahead and start.